test/cmptestlib.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2007-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Nokia 2007-2018
+ * Copyright Siemens AG 2015-2018
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ * Copyright OpenSSL 2007-2018
+ * Copyright Nokia 2007-2018
+ * Copyright Siemens AG 2015-2018
+ *
+ * Contents licensed under the terms of the Apache License 2.0
+ * See https://www.openssl.org/source/license.html for details
+ *
+ * CMP tests by Martin Peylo, Tobias Pankert, and David von Oheimb.
+ */
+
+#include "cmptestlib.h"
+#include <openssl/rsa.h> /* needed in case config no-deprecated */
+
+EVP_PKEY *load_pem_key(const char *file)
+{
+    EVP_PKEY *key = NULL;
+    BIO *bio = NULL;
+
+    if (!TEST_ptr(bio = BIO_new(BIO_s_file())))
+        return NULL;
+    /* BIO_new_file(file, "r"); */
```

Comment:
Can this be removed?

Reply:
Yes, this comment can be removed. 
Fixed.